Installing-Oppia.md

```diff
@@ -1,455 +1,35 @@
-## Table of Contents
+__If you are looking for more elaborate instructions on how to get started with Oppia, go to [setting things up](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up).__
+If you get stuck somewhere and need help please refer to our [[Getting Help Page|Get-help]].
 
-* [Install prerequisites](#install-prerequisites)
-* [Clone Oppia](#clone-oppia)
-* [Setup a virtual environment](#setup-a-virtual-environment)
-* [Running Oppia on a development server](#running-oppia-on-a-development-server)
-* [Tips and tricks](#tips-and-tricks)
-* [Notes on installation on Arch Linux systems](#notes-on-installation-on-arch-linux-systems)
-  * [Changes to installation prerequisites](#changes-to-installation-prerequisites)
-  * [Changes to the virtual environment setup](#changes-to-the-virtual-environment-setup)
+Note that throughout our wiki, we use the following notations:
 
-**Note:** If you just want to create and share explorations, you may be able to use the hosted server at https://www.oppia.org (in which case you don't need to install anything).
```

Comment:
I think we should keep this?

Reply:
Sorry, you can ignore this comment. It was made on an older version of this PR.

Reply:
Done!